content/en/dashboards/widgets/powerpack.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Powerpack Widget
+kind: Documentation
+disable_toc: false
+further_reading:
+- link: "/dashboards/guide/powerpacks-best-practices/"
+  tag: "Guide"
+  text: "Scale Graphing Expertise with Powerpacks"
+- link: "https://www.datadoghq.com/blog/standardize-dashboards-powerpacks-datadog/"
+  tag: "Blog"
+  text: "Save dashboard widgets in reusable groups with Powerpacks"
+- link: "/dashboards/widgets/group/"
+  tag: "Documentation"
+  text: Group Widget
+- link: "/dashboards/graphing_json/"
+  tag: "Documentation"
+  text: "Building Dashboards using JSON"
+---
+
+## Overview
+
+<div class="alert alert-info">Powerpack widgets are not supported on screenboards.</div>
+
+Powerpacks are templated groups of widgets that scale graphing expertise as reusable dashboard building blocks. Powerpacks are either preset (created by Datadog, available to all customers) or custom (created by a user, available only within their organization). For more information on Powerpack best practices, see the [Scale Graphing expertise with Powerpacks][1] guide.
+
+## Setup
+
+### Creating a Powerpack
+
+Create a Powerpack from an existing group on a dashboard: 
+
+{{< img src="dashboards/widgets/powerpack/group_header_icons.png" alt="Dashboard group header icons highting the Save as Powerpack icon option" style="width:80%;" >}}
+
+1. From a dashboard group's header, click the "Save as Powerpack" icon. 
+1. Fill out details to make this powerpack discoverable to your organization. 
+1. Add tags under "Add Search Categories" to organize your Powerpacks. This allows team members to find the correct Powerpack to add to their dashboard.
+1. Choose which filters should be configurable for the users of this powerpack. 
```

Comment:
There are also some inconsistencies on how you capitalize powerpacks throughout both files in this PR. Here it really goes back and fourth in every other step.